common/cultures/rathnir_cultures.txt

```diff
@@ -366,4 +390,39 @@ luthic_group = {
 
 	}
 }
-	
\ No newline at end of file
+
+arcadian_group = {
+	graphical_culture = polynesiangfx
+
+	astanan = {
+		primary = A43 #Astana

```

Comment:
Astana is the capital city of Kazakhstan, we retconned it to something else, can't find it rn

Reply:
Agriana; province ID: 387. Source: Mistic

Reply:
can strike these in main, the cultures are now defunct